### ISSUE TITLE
parallel entry

### ISSUE THREAD

**Issue by nicholasjhorton** [hglanz/DS-Blog-Ex#80]
I ran into problems building the site so I had to disable the draft entry (see https://github.com/hglanz/DS-Blog-Ex/blob/master/content/post/2019_07_12_Parallel/parallel.disabled). I will assign this to myself to figure out what I have to do on my end.

**Comment by nicholasjhorton** [comment]
I fixed some typos (embarrassing was misspelled throughout) and did a rewrite of the first paragraph (see https://github.com/hglanz/teachdatascience_blog/commit/631cfd83cdcc1b683a87d02497937988a05f350d).  I also added a plug for the HDSR ambitious data science paper and Sakr's CMU course.  I'm available to talk this morning if you see any issues or problems.  Again, sorry for the delay in reviewing this.
